sale_product_deposit/__manifest__.py

```diff
@@ -13,7 +13,7 @@
     "author": "BEES coop - Cellule IT, Coop IT Easy SC",
     "website": "https://github.com/beescoop/Obeesdoo",
     "category": "Sales",
-    "version": "12.0.1.0.0",
+    "version": "12.0.2.0.0",
```

Comment:
```suggestion
    "version": "12.0.1.0.1",
```
i would suggest a patch release for this also. (this requires to also rename the migration script.)

Reply:
done